pkg/controller/constraint/constraint_controller.go

```diff
@@ -50,7 +57,44 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller").WithValues(logging.Process, "constraint_controller")
+var log = logf.Log.V(logging.DebugLevel).WithName("controller").WithValues(logging.Process, "constraint_controller")
```

Comment:
Are there any log lines that we'd miss by shifting all logs down to debug level?

Reply:
All log lines were either already using Info (so this would put the logging level in the right place) and/or returning error to the caller, which would be logged by the caller.